src/libs/actions/Report.js

```diff
@@ -582,6 +582,19 @@ function readOldestAction(reportID, reportActionID) {
     );
 }
 
+/**
+ * Gets metadata info about links in the provided report
```

Comment:
I think this is linked to the report action firstly
```suggestion
 * Gets metadata info about links in the provided report action
```